src/knightos/display.c

```diff
@@ -119,16 +119,16 @@ void draw_v_line(SCREEN *screen, unsigned char x, unsigned char y, unsigned char
 
 void draw_sprite(SCREEN *screen, unsigned char x, unsigned char y, unsigned char height, const void *sprite, GFX_BLEND_MODE blend_mode) {
 	switch (blend_mode) {
-	case GFX_AND:
-		draw_sprite_and(screen, x, y, height, sprite);
-    	break;
-	case GFX_OR:
-		draw_sprite_or(screen, x, y, height, sprite);
-    	break;
- 	case GFX_XOR:
-		draw_sprite_xor(screen, x, y, height, sprite);
-    	break;
-    }
+		case GFX_AND:
```

Comment:
Sorry - keep the case aligned with the switch. Just bring the break up an indent.